accio-sqlrewrite/src/test/java/io/accio/sqlrewrite/TestModel.java

```diff
@@ -0,0 +1,146 @@
+                                "select * from main.orders",
+                                List.of(
+                                        column("orderkey", INTEGER, null, true),
+                                        column("custkey", INTEGER, null, true),
+                                        column("orderstatus", VARCHAR, null, true),
+                                        column("totalprice", INTEGER, null, true),
+                                        column("orderdate", DATE, null, true),
+                                        column("orderpriority", VARCHAR, null, true),
+                                        column("clerk", VARCHAR, null, true),
+                                        column("shippriority", INTEGER, null, true),
+                                        column("comment", VARCHAR, null, true),
+                                        column("lineitem", "Lineitem", "OrdersLineitem", true)),
+                                "orderkey"),
+                        model("Lineitem",
+                                "select * from main.lineitem",
+                                List.of(
+                                        column("orderkey", INTEGER, null, true),
+                                        column("partkey", INTEGER, null, true),
+                                        column("suppkey", INTEGER, null, true),
+                                        column("linenumber", INTEGER, null, true),
+                                        column("quantity", INTEGER, null, true),
+                                        column("extendedprice", INTEGER, null, true),
+                                        column("discount", INTEGER, null, true),
+                                        column("tax", INTEGER, null, true),
+                                        column("returnflag", VARCHAR, null, true),
+                                        column("linestatus", VARCHAR, null, true),
+                                        column("shipdate", DATE, null, true),
+                                        column("commitdate", DATE, null, true),
+                                        column("receiptdate", DATE, null, true),
+                                        column("shipinstruct", VARCHAR, null, true),
+                                        column("shipmode", VARCHAR, null, true),
+                                        column("comment", VARCHAR, null, true),
+                                        column("orderkey_linenumber", VARCHAR, null, true, "concat(orderkey, '-', linenumber)")),
+                                "orderkey_linenumber")))
+                .setRelationships(List.of(
+                        relationship("OrdersCustomer", List.of("Orders", "Customer"), MANY_TO_ONE, "Orders.custkey = Customer.custkey"),
+                        relationship("OrdersLineitem", List.of("Orders", "Lineitem"), MANY_TO_ONE, "Orders.orderkey = Lineitem.orderkey")))
```

Comment:
Orders and Lineitem are ONE_TO_MANY relationship. Could you help me to fix the relationship in `TestMetric`? It's my fault.
```suggestion
                        relationship("OrdersLineitem", List.of("Orders", "Lineitem"), ONE_TO_MANY, "Orders.orderkey = Lineitem.orderkey")))
```